app/serializers/playlist_serializer.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: playlists
+#
+#  id            :bigint           not null, primary key
+#  description   :string
+#  name          :string           not null
+#  playlist_type :integer          not null
+#  private       :boolean          default(FALSE)
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#  user_id       :bigint
+#
+class PlaylistSerializer < ActiveModel::Serializer
+  attributes :id, :name, :description, :user_id, :playlist_type, :created_at, :updated_at, :item_ids, :private
```

Comment:
Have you tested what happens with a huge amount of `item_ids`?

Reply:
I just did a few test. There was some N+1 querying going on, but this is now solved.

When we have multiple playlists with a total of 60000 items between them, the request takes between 800ms and 1000ms. This is certainly something to watch out for, if a request would need to return the full 500 lists, all with a >1000 items in each one, this request would probably be painfully slow.

Maybe we need to use a lower number of playlists per page?

Reply:
I added an extra commit, which improves this situation A LOT. The same request with 60000 playlist_items now takes 170 - 300ms and should be able to scale without much issues. It does require more (postgres-only) SQL to be present in the model, but I'd say that is worth it here?